docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -198,6 +199,110 @@ SELECT levenshtein('Yugabyte', 'yugabyte'), metaphone('yugabyte', 8);
 (1 row)
 ```
 
+### HypoPG example
+
+```sql
+CREATE EXTENSION hypopg;
+CREATE TABLE up_and_down (up int primary key, down int);
+INSERT INTO up_and_down SELECT a AS up, 10001-a AS down FROM generate_series(1,10000) a;
+```
+
+The `up_and_down` table has no indexes, but because it's defined with a primary key, it is ordered by, and the records can be directly retrieved using, the primary key:
```

Comment:
```suggestion
The `up_and_down` table has no indexes, but because it's defined with a primary key, it is ordered by, and the records can be directly retrieved using the primary key:
```